intro-sql-access.mdx

```diff
@@ -0,0 +1,158 @@
+
+You can find more details about the SQL access in the documentation.
+
+### Xata rich types
+
+As you access your database directly using SQL, the Xata rich column types (File, email) have reduced functionality. For example, the File type are currently read-only when accessed over SQL. You can only add or edit some of the rich types using the Xata API.
+
+## Examples and SDK support
+
+SQL is supported in both the TypeScript/Javascript and the Python SDKs, as well as via the REST API. In it’s simple form, you can run SQL like this:
+
+<TabbedCode tabs={['TypeScript', 'Python', 'REST']}>
+
+```ts
+const { records: myRecord } = await xata.sql<TeamsRecord>`SELECT * FROM "teams"`;
+```
+
+```python
+records = xata.sql().query('''SELECT * FROM "teams"''')
+```
+
+```json
+// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/sql
+{
+  "statement": "SELECT * FROM \"teams\""
+}
+```
+
+</TabbedCode>
+
+Currently, each request can contain a single SQL statement. You can submit parameterized queries with parameters, which you should use to protect against SQL injection. Here is an example:
+
+<TabbedCode tabs={['TypeScript', 'Python', 'REST']}>
+
+```tsx
+const { records: myRecord } = await xata.sql<TeamsRecord>`SELECT * FROM "teams" WHERE name=${name}`;
+```
+
+```python
+records = xata.sql().query('''SELECT * FROM "teams" WHERE name=$1''', params=[name])
+```
+
+```json
+// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/sql
+{
+  "statement": "SELECT * FROM \"teams\" WHERE name=$",
+  "params": ["name"]
+}
+```
+
+</TabbedCode>
+
+The above examples, however, could be easier achieved using the Xata API, and we generally recommend you using the Xata SDKs for this types of queries because they provide better type safety-ness (in TypeScript) and overall a better developer experience.
+
+SQL, on the other hand, is useful as an escape-hatch for features that are not available (yet) in the Xata API. For example, the following uses the `md5` and `avg` PostgreSQL functions and groups the results by the title:
+
+<TabbedCode tabs={['TypeScript', 'Python', 'REST']}>
+
+```tsx
+const { records: myRecord } =
+  await xata.sql<TeamsRecord>`SELECT "primaryTitle", md5("primaryTitle"), avg("runtimeMinutes") FROM titles GROUP BY "primaryTitle"`;
+```
+
+```python
+records = xata.sql().query('''SELECT "primaryTitle", md5("primaryTitle"), avg("runtimeMinutes") FROM titles GROUP BY "primaryTitle"''')
+```
+
+```json
+// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/sql
+{
+  "statement": "SELECT \"primaryTitle\", md5(\"primaryTitle\"), avg(\"runtimeMinutes\") FROM titles GROUP BY \"primaryTitle\""
+}
+```
+
+</TabbedCode>
+
+In the above example, note that the double-quotes are required because the column names are in camelCase.
```

Comment:
We use "In the above example" in back to back sentences. Attempting to remove the duplication in this suggestion:

"It's worth noting that the double-quotes are required because the column names are in camelCase."